docs/advanced-topics/checker-workflows.rst

```diff
@@ -98,7 +98,7 @@ parameter.
 
 Recall that the checker workflow refers to the workflow that connects
 the original entry with the validation tool/workflow, so that it can be
-run as one workflow. The validation tool/workflow is what does the bulk
+run as one workflow. The validation tool/workflow is what does the actual
```

Comment:
"actual of the validation" -> "actual validation".

Reply:
I... don't know how I missed that, lol. Thanks for catching.